README.md

```diff
@@ -88,6 +94,9 @@ def catcher_SIGINT(t: ThreadContext, catcher: SignalCatcher) -> None:
 def catcher_SIGPIPE(t: ThreadContext, catcher: SignalCatcher) -> None:
     print(f"SIGPIPE: Signal number {catcher}")
 
+def handle_geteuid(t: ThreadContext, handler: SyscallHandler) -> None:
```

Comment:
handler*

Reply:
Addressed in [9e4a7f8](https://github.com/libdebug/libdebug/pull/129/commits/9e4a7f831a709f7ca168593df7deb81678b86ed5)